magento/magento2
# [Shipping] Adding the tracking number client validation

### Description (*)
This PR provides a client validation for tracking number.

### Fixed Issues (if relevant)
1. magento/magento2#24745: Magento pop up not display on tracking info message

### Manual testing scenarios (*)
1. Create a new order
2. Create a shipment for that order
3. Navigate to the shipment
4. Add a new tracking carrier
5. The Tracking Number should be validated before sending the request
![image](https://user-images.githubusercontent.com/15868188/66028572-16682800-e506-11e9-8ffb-80c64d69ac18.png)


### Questions or comments


### Contribution checklist (*)
